{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2018-20458",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://github.com/radare/radare2/commit/4e98402f09a0ef0bb8559a33a4c1988c54938eaf",
          "name": "https://github.com/radare/radare2/commit/4e98402f09a0ef0bb8559a33a4c1988c54938eaf",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://github.com/radare/radare2/issues/12374",
          "name": "https://github.com/radare/radare2/issues/12374",
          "refsource": "MISC",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In radare2 prior to 3.1.1, r_bin_dyldcache_extract in libr/bin/format/mach0/dyldcache.c may allow attackers to cause a denial-of-service (application crash caused by out-of-bounds read) by crafting an input file."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-12-25T19:29Z",
  "lastModifiedDate": "2018-12-25T19:29Z"
}